scripts/userstyles.yml

```
@@ -770,6 +770,13 @@ userstyles:
       app-link: "https://reddit.com"
     current-maintainers: []
     past-maintainers: [*jayylmao, *rubyowo]
+  rentry:
+    name: Rentry
```

```suggestion
  rentry.co:
    name: Rentry.co
```
This is how they brand themselves from what I can tell. Make sure to update the UserCSS metadata and directory name.